kinto/config/kinto.tpl

```diff
@@ -78,9 +159,80 @@ multiauth.policies = basicauth
 # fxa-oauth.relier.enabled = true
 # fxa-oauth.webapp.authorized_domains = *
 
+# Plugins
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#plugins
+# https://github.com/uralbash/awesome-pyramid
+kinto.includes = kinto.plugins.default_bucket
+#                kinto.plugins.admin
+#                kinto.plugins.accounts
+#                kinto.plugins.history
+#                kinto.plugins.quotas
+
+# Pluggable components
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#pluggable-components
 #
-# Client cache headers
+# kinto.logging_renderer = your_log_renderer.CustomRenderer
+
+# Notifications
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#notifications
+#
+# Configuration example:
+# kinto.event_listeners = redis
+# kinto.event_listeners.redis.use = kinto_redis.listeners
+# kinto.event_listeners.redis.url = redis://localhost:6379/0
+# kinto.event_listeners.redis.pool_size = 5
+# kinto.event_listeners.redis.listname = queue
 #
+# Setting up notifications via WebSocket
+# https://kinto.readthedocs.io/en/latest/tutorials/notifications-websockets.html#tutorial-notifications-websockets
+
+# Filtering
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#filtering
+#
```

Comment:
I don't think this deserves a dedicated «section/title». It should be moved along configuration example